README.md

```diff
@@ -13,3 +13,15 @@ A Roc [platform](https://github.com/roc-lang/roc/wiki/Roc-concepts-explained#pla
   - [0.9.x](https://www.roc-lang.org/packages/basic-cli/0.9.1)
   - [0.8.x](https://www.roc-lang.org/packages/basic-cli/0.8.1)
   - [latest main branch](https://www.roc-lang.org/packages/basic-cli)
+
+## Running locally
+
+If you clone this repo instead of using the release URL you'll need to build the platform once:
+```sh
+roc build.roc
```

Comment:
Should we make this `roc build.roc --release` @lukewilliamboswell?

Reply:
Well, that gives a debug build. I figured that's a good default like rust with just `cargo build`. But tbh I didn't put a lot of thought into it.

Reply:
I was hitting an error without --release, so I added it in the README.

Reply:
I've now removed the --release flag, because it does not work without it and users would see a complicated error simply because they forgot to add `--release`.